nvidia-merged/PKGBUILD

```diff
@@ -290,6 +291,8 @@ package_nvidia-merged-utils() {
     install -D -m644 "${_vgpupkg}/init-scripts/systemd/nvidia-vgpu-mgr.service" "${pkgdir}/usr/lib/systemd/system/nvidia-vgpu-mgr.service"
     install -D -m644 "${srcdir}/vgpu_unlock-rs.conf" "${pkgdir}/usr/lib/systemd/system/nvidia-vgpud.service.d/30-vgpu_unlock-rs.conf"
     install -D -m644 "${srcdir}/vgpu_unlock-rs.conf" "${pkgdir}/usr/lib/systemd/system/nvidia-vgpu-mgr.service.d/30-vgpu_unlock-rs.conf"
+    install -D -m644 "${srcdir}/unlock-profile_override.toml" "${pkgdir}/etc/vgpu_unlock/profile_override.toml"
+    install -D -m644 "${srcdir}/unlock-config.toml" "${pkgdir}/etc/vgpu_unlock/config.toml"
```

Comment:
The config file is for turning migration on.

Reply:
Most of above still holds true. 

Additionally, if migration were a feature that should be enabled globally, it would be set that way in vgpu_unlock-rs, akin to how `unlock` defaults to true unless you explicitly disable it.

Given that `unlock_migration` defaults to false, I see no reason to force an opinionated default in place of what is built into the library.